tests/test_component.py

```diff
@@ -78,10 +36,8 @@ def test_methods(self):
         assert_array_equal(model, truth)
         assert_array_equal(other_model, other_truth)
 
-        with pytest.raises(ValueError):
-            component.get_model(sed)
-        with pytest.raises(ValueError):
-            component.get_model(morph=morph)
+
+        # TODO: test component with partial parameter list
```

Comment:
Typically on LSST anytime we place a `TODO` in the code we open a ticket and put the ticket number in the comment. I think that's a good practice for us to adopt, so it might be worth opening an issue and writing the issue number below the `TODO` comment.